frontend/src/dashboard.js

```diff
@@ -328,7 +393,7 @@ export class Dashboard extends React.Component {
             onChange={this.onTextInputChange}
             id="typeahead-select-input"
             autoComplete="off"
-            placeholder={dashboard ? dashboard.title : "No active dashboard"}
+            placeholder={activeDashboard?.title || "Getting default dashboard..."}
```

Comment:
I think we need to leave here `"No active dashboard"` placeholder as it clearly says that if there is no dashboard selected and no input value - it means we don't have dashboard selected.